code/game/machinery/autolathe.dm

```diff
@@ -1,444 +1,447 @@
-#define AUTOLATHE_MAIN_MENU       1

```

Comment:
You seem to have messed up line endings here

Reply:
wahooooooooooooooooooooooooooooooooooooooooooo